setup.py

```diff
@@ -1,9 +1,20 @@
 from setuptools import setup, find_packages
 
+# We define the version number in restructuredtext_lint/__init__.py
+# Here we can't use "import restructuredtext_lint" to access it
+# as "restructuredtext_lint.__version__" as that would tell us the
+# previously installed version (if any).
+__version__ = None
+for line in open('restructuredtext_lint/__init__.py'):
+    if (line.startswith('__version__ = ')):
+        exec(line.strip())
+if __version__ is None:
+    import sys
+    sys.exit("Internal error reading version number.")
```

Comment:
Lint checks to enforce single vs double quotes - nice idea. Although personally I'd go for double quotes ;)